dataedit/templates/dataedit/dataview.html

```diff
@@ -117,14 +116,21 @@ <h4>API Usage</h4>
   # Process the row
   </textarea>
   <hr>
+
+  <h4>Permissions</h4>
+  <div class="btn-group btn-group-sm" role="group" aria-label="Actions">
+    <a href="{{ table }}/permissions" type="button" class="btn btn-danger">Permissions</a>
+  </div>
+  <hr>
+
   <h4>
     Meta Information
   </h4>
-  <div class="btn-group btn-group-sm">
-    <button class="btn btn-success" onclick="downloadMetadata();">Download JSON</button>
+  <div class="mb-2">
+    <button class="btn btn-sm btn-success" onclick="downloadMetadata();">Download JSON</button>
```

Comment:
```suggestion
    <button class="btn btn-sm btn-success" onclick="downloadMetadata();" data-toggle="tooltip" title="Download metadata in JSON format">Download JSON</button>
```

Reply:
We can add the `not can_edit` conditional here, too. But I feel that a tooltip could improve this button in general.